tests/lib/external/snapd-testing-tools/utils/log-parser

```diff
@@ -96,34 +63,33 @@ class Result:
 
     def __repr__(self):
         if self.detail:
-            return '{}{}'.format(self.source_line, str(self.detail))
+            return "{}{}".format(self.source_line, str(self.detail))
         return self.source_line
 
     def __dict__(self):
         prepared_detail = None
         if self.detail:
             prepared_detail = self.detail.__dict__()
         return {
-            'type': self.type,
-            'date': self.date,
-            'time': self.time,
-            'result_type': self.result_type,
-            'level': self.level,
-            'stage': self.stage,
-            'number': self.number,
-            'detail': prepared_detail
-            }
+            "type": self.type,
+            "date": self.date,
+            "time": self.time,
+            "result_type": self.result_type,
+            "level": self.level,
+            "stage": self.stage,
+            "number": self.number,
+            "detail": prepared_detail,
+        }
 
 
 class Info:
     """
-    Info represents the extra tasks information which is included in the
-    spread log. The info can be: Error, Debug and Warning
+    Info represents the tasks status information which is included
+    in the spread log. The info can be: Error, Debug and Warning.
     """
 
-    def __init__(self, info_type, verb, task, extra, date, time,
-                 detail, source_line):
-        self.type = INFO
+    def __init__(self, info_type, verb, task, extra, date, time, detail, source_line):
```

Comment:
@zyga wrote: There's more confusing undocumented arguments here: what is extra for?

Please document it.